frontend/simple/components/CreateGroup/CreateGroupSummary.vue

```diff
@@ -0,0 +1,110 @@
+<template>
+  <div>
+    <h1 id="summaryStep" class="title is-1 has-text-centered"><i18n>Review & Finish</i18n></h1>
+
+    <h2 class="title is-6"><i18n>Group name:</i18n></h2>
+    <p class="subtitle is-2" :class="!group.groupName && 'is-danger'">{{ group.groupName || 'No group name set' }}</p>
+    <p class="has-text-right">
+      <router-link
+        class="button is-primary"
+        :to="{name: 'CreateGroupName'}"
+      >
+        <i18n>Edit</i18n>
+      </router-link>
+    </p>
+    <hr>
+
+    <h2 class="title is-6"><i18n>Group purpose:</i18n></h2>
+    <p class="subtitle is-2" :class="!group.sharedValues && 'is-danger'">{{ group.sharedValues || 'No group purpose set' }}</p>
```

Comment:
The font here is too big for the group purpose to be displayed (which can be quite long potentially).

Also, don't forget that the error text here needs to be localized (see comment above).

Honestly, I think that, as per the comments elsehwere, we shouldn't even have the option of saying `'No group purpose set'`, it should just always be set by the time this view is displayed, IMO.